### PR TITLE
information added for how to reinstall triggers

### PR DESCRIPTION
**Pull request by konarshankar07**
## Purpose of this pull request

This pull request (PR) provide information about how can you reinstall triggers in mysql

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-index.html#configure-indexers-1

Fixed https://github.com/magento/devdocs/issues/4647

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/config-guide/cli/config-cli-subcommands-index.md [hunk 1]
```diff
@@ -120,6 +120,20 @@ Product Price index has been rebuilt successfully in <time>
 Catalog Search index has been rebuilt successfully in <time>
 ```
 
+To reinstall triggers:
```
Comment (rogyar) [inline guides/v2.3/config-guide/cli/config-cli-subcommands-index.md]: Hi @konarshankar07. Thank you for providing the information. 
Currently, as a reader, I find this information a little bit confusing. I'm not sure what's a trigger, why I need to rebuild the triggers? So we a little bit out of context here. 

I would suggest extending the [following](https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-index.html#configure-indexers-1) section with a mention that
> The indexers-related database triggers are being added when the indexer mode is set to `schedule` and removed when the indexer mode is set to `realtime`.

Thank you!

Comment (konarshankar07) [reply guides/v2.3/config-guide/cli/config-cli-subcommands-index.md]: Hello @rogyar ,
I have updated the content as per your above feedback. My question is how we can reference this in MySQL section which is mention in the task?
Thanks

**Comment by hostep** [comment]
Thanks for taking this on @konarshankar07!

Looks good so far.
As for your question where to reference it from the [mysql section](https://devdocs.magento.com/guides/v2.3/install-gde/prereq/mysql.html#instgde-prereq-mysql-intro), maybe something like this:

Magento uses MySQL database triggers to improve database access during reindexing. **These get created when [the indexer mode is set to `schedule`](https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-index.html#configure-indexers-1).** Magento does not support any custom tr...

The part in bold. This is just a suggestion 🙂

**Comment by konarshankar07** [comment]
Hello @hostep ,
Thanks for the feedback. I'll update the changes for mention feedback but still as a user point of view its very difficult to find exact information which they are looking for like "reinstall triggers or reinstall MySQL triggers" might the be keywords user will search which is currently missing.

**Comment by hostep** [comment]
@konarshankar07 

> but still as a user point of view its very difficult to find exact information which they are looking for like "reinstall triggers or reinstall MySQL triggers" might the be keywords user will search which is currently missing.

Maybe we should add this additional line then:
> The indexers-related database triggers are being added when the indexer mode is set to `schedule` and removed when the indexer mode is set to `realtime`.
**If the triggers are missing somehow from your database while the indexers are set to `schedule`, then a way to reinstall them is by first changing the indexers to `realtime` and afterwards changing them back to `schedule`. That will reinstate them.**

Just a suggestion again 🙂

**Comment by konarshankar07** [comment]
Thanks @rogyar and @hostep for the feedbacks.

**Comment by dobooth** [comment]
running tests

**Pull request merged**